src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/dto/trust/TrustIndividualDto.java

```diff
@@ -288,17 +324,24 @@ public void setUraAddressRegion(String uraAddressRegion) {
     }
 
     public AddressDto getUsualResidentialAddress() {
-        var address = new AddressDto();
-        address.setPropertyNameNumber(uraAddressPremises);
-        address.setLine1(uraAddressLine1);
-        address.setLine2(uraAddressLine2);
-        address.setCounty(uraAddressRegion);
-        address.setLocality(uraAddressLocality);
-        address.setCountry(uraAddressCountry);
-        address.setCareOf(uraAddressCareOf);
-        address.setPoBox(uraAddressPoBox);
-        address.setPostcode(uraAddressPostalCode);
-        return address;
+        // If the address is being set from the ura fields generate the address
```

Comment:
Suggest making this comment a bit more explicit, as discussed. E.g.

"When converting from DTO to DAO the individual address fields will be present and need to be converted to an address object during the mapping process. When converting from DAO to DTO the individual fields will not be populated and so the address object just needs to be returned"

Probably then just need to duplicate this comment in all the places that this is happening. Four?

As we all agreed, a better and consistent interface between web and API for the trust data would be the best fix for this...